core/broadleaf-framework/src/main/java/org/broadleafcommerce/core/catalog/domain/CategoryImpl.java

```diff
@@ -231,7 +232,12 @@ private static void fillInURLMapForCategory(Map<String, List<Long>> categoryUrlM
 
     @Column(name = "ACTIVE_END_DATE")
     @AdminPresentation(friendlyName = "CategoryImpl_Category_Active_End_Date", order = 2000,
-            group = Presentation.Group.Name.ActiveDateRange, groupOrder = Presentation.Group.Order.ActiveDateRange)
+            group = Presentation.Group.Name.ActiveDateRange, groupOrder = Presentation.Group.Order.ActiveDateRange,
+            validationConfigurations = { @ValidationConfiguration(
+                    validationImplementation = "blAfterStartDateValidator",
+                    configurationItems = {
+                            @ConfigurationItem(itemName = "otherField", itemValue = "activeStartDate")
+                    }) })
```

Comment:
Separate the different `})` on different lines. The second one should be at the same indentation level as `validationConfigurations`